### DISCUSSION TITLE
Question: how did you find BarTranslate?

### DISCUSSION THREAD

**Discussion by ThijmenDam** [ThijmenDam/BarTranslate#32] [General]
Just out of curiosity, how did you come across BarTranslate? 

This application began as a small personal endeavour to assist me in writing my Master's thesis. However, it has since gained some traction on GitHub, and I'm interested in knowing how you found BarTranslate.

**Comment by danilo-sa** [comment]
Hi there,

I found your development googling about some feature like yours.

Congrants for your work.

Can I propose some improvements? 😊

Make possible perform changes in the tool window size.
Add features to de configuration, like run during the startup. (not a problem, I made the change in Settings)

Overall the solution is amazing.

**Comment by SarvagyaVaish** [comment]
I was looking for a translation widget and googled "mac translate widget".
